rainier-core/src/main/scala/com/stripe/rainier/core/SBC.scala

```diff
@@ -22,9 +22,11 @@ final case class SBC[T, L <: Distribution[T]](priors: Seq[Continuous],
   def posteriorSamples(nSamples: Int)(implicit rng: RNG): List[T] =
     RandomVariable
       .traverse(priors.map(_.param))
-      .map { priorParams =>
+      .flatMap { priorParams =>
```

Comment:
@avi-stripe addressing your comment about exercising the logDensity and generator for both the prior and likelihood: I'm almost doing `model(1000).sample(nSamples)`, but `SBC.fit` returns the prior so I did this to get samples from the posterior. It looks like we've got everything now because we use both generators in `synthesize` (and the posterior generator again here) and flatmapping with a call to `RandomVariable.fit` uses both logDensities. This is pretty similar to `fit` and `synthesize` so lemme know if you see a better way to do this.

Reply:
why do we care, though? can't we just use the (inferred) prior parameter as our output?

Reply:
Oh, I see, since we're fitting now, that incorporates information about the likelihood so we'll get different prior params even for models with the same prior distribution. Yeah, that's a good point. I'll try switching it.